src/Peering/Peering/help/New-AzPeeringExchangeConnectionObject.md

```diff
@@ -0,0 +1,206 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Peering.dll-Help.xml
+Module Name: Az.Peering
+online version: https://docs.microsoft.com/en-us/powershell/module/az.peering/new-azpeeringexchangeconnectionobject
+schema: 2.0.0
+---
+
+# New-AzPeeringExchangeConnectionObject
+
+## SYNOPSIS
+Creates a in memory PSObject to be used for creating or modifying a Peering.
+
+## SYNTAX
+
+### ParameterSetNameIPv4Address (Default)
+```
+New-AzPeeringExchangeConnectionObject [-PeeringDBFacilityId] <Int32> -PeerSessionIPv4Address <String>
+ [-MaxPrefixesAdvertisedIPv4 <Int32>] [-MD5AuthenticationKey <String>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ParameterSetNameIPv6Address
+```
+New-AzPeeringExchangeConnectionObject [-PeeringDBFacilityId] <Int32> -PeerSessionIPv6Address <String>
+ [-MaxPrefixesAdvertisedIPv6 <Int32>] [-MD5AuthenticationKey <String>]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ParameterSetNameIPv4AddressParameterSetNameIPv6Address
+```
+New-AzPeeringExchangeConnectionObject [-PeeringDBFacilityId] <Int32> -PeerSessionIPv4Address <String>
+ -PeerSessionIPv6Address <String> [-MaxPrefixesAdvertisedIPv4 <Int32>] [-MaxPrefixesAdvertisedIPv6 <Int32>]
+ [-MD5AuthenticationKey <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Creates an in memory PSObject 
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:> $exconnection = New-AzPeeringExchangeConnectionObject -PeeringDBFacilityId 99999 -PeerSessionIPv4Address 10.3.151.99 -MaxPrefixesAdvertisedIPv4 20000 -MD5AuthenticationKey 25234523452123411fd234qdwfas3234
+
+PeeringDBFacilityId     : 99999
+PeerSessionIPv4Address  : 10.3.151.99
+MaxPrefixesAdvertisedV4 : 20000
+Md5AuthenticationKey    : 25234523452123411fd234qdwfas3234
+```
+
+Local connection object
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -MaxPrefixesAdvertisedIPv4
+HelpMaxAdvertisedIPv4
+
+```yaml
+Type: System.Nullable`1[System.Int32]
+Parameter Sets: ParameterSetNameIPv4Address, ParameterSetNameIPv4AddressParameterSetNameIPv6Address
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -MaxPrefixesAdvertisedIPv6
+HelpMaxAdvertisedIPv6
```

Comment:
Fix the spacing for these help messages everywhere - you will need to do this manually as the help file is not automatically updated when the cs file help messages are updated.